CMSIS/DoxyGen/Core/src/Template.txt

```diff
@@ -377,6 +378,73 @@ The files for other compiler vendors differ slightly in the syntax, but not in t
 \verbinclude "Source/ARM/startup_Device_ac6.S"
 */
 
+/*=======0=========1=========2=========3=========4=========5=========6=========7=========8=========9=========0=========1====*/
+/**
+\page linker_sct_pg Scatter-Loading description file <device>_ac<5|6>.sct
+
+If C-Startup is used then it is necessary that also a scatter file for linking is used.
```

Comment:
A scatter file for linking is required when using a \ref startup_c_pg "C startup file".

Reply:
changed.